README.md

```diff
@@ -38,7 +47,7 @@ steps:
     plugins:
       - cultureamp/aws-assume-role:
           role: ${BUILD_ROLE}
-      - cultureamp/ecr-scan-results#v1.1.7:
+      - cultureamp/ecr-scan-results#v1.2.0:
           image-name: "$BUILD_REPO:deploy-$BUILD_TAG"
           max-criticals: "1"
           max-highs: "10"
```

Comment:
`indent -= 1`

Reply:
There are different styles for YAML formatting, but I double-checked this just now and it's at the least not wrong...

Reply:
Yeah you're right... looks fine in my editor, but in the GitHub diff it's off by one space:

<img width="622" alt="image" src="https://user-images.githubusercontent.com/4476568/161686071-0d898986-df87-4d52-b06f-d0e327eaa8c7.png">

Reply:
That is definitely an odd effect 😵‍💫